.github/workflows/sync-universe-param-files.yaml

```
@@ -0,0 +1,26 @@
+name: sync-universe-param-files
+
+on:
+  schedule:
+    - cron: 0 0 * * *
+  workflow_dispatch:
+
+jobs:
+  sync-universe-param-files:
```

It's not limited to only `universe`.

```suggestion
name: sync-param-files

on:
  schedule:
    - cron: 0 0 * * *
  workflow_dispatch:

jobs:
  sync-param-files:
```

---

https://github.com/autowarefoundation/autoware_launch/pull/64/commits/a72531e97e0a0d1b1f1fae0a7eb4081cc311abae
DONE